XamarinCommunityToolkit/Views/AvatarView/ColorThemes.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public struct ColorThemes
+	{
+		public static IColorTheme Default
```

Comment:
I propose to convert these computed properties to read-only properties or fields

Reply:
And I think all they should belong to ColorTheme class (Similar to Easing class in forms)